src/backersManager/newAllocation.ts

```diff
@@ -1,10 +1,21 @@
-import { NewAllocation as NewAllocationEvent } from "../generated/BackersManagerRootstockCollective/BackersManagerRootstockCollective";
-import { BackerStakingHistory, GaugeStakingHistory } from "../generated/schema";
-import { BackersManagerRootstockCollective as BackersManagerRootstockCollectiveContract } from "../generated/BackersManagerRootstockCollective/BackersManagerRootstockCollective";
+import { NewAllocation as NewAllocationEvent } from "../../generated/BackersManagerRootstockCollective/BackersManagerRootstockCollective";
+import {
+  BackerStakingHistory,
+  GaugeStakingHistory,
+  Backer,
+  BackerToBuilder,
+  GaugeToBuilder,
+} from "../../generated/schema";
+import { BackersManagerRootstockCollective as BackersManagerRootstockCollectiveContract } from "../../generated/BackersManagerRootstockCollective/BackersManagerRootstockCollective";
 import { BigInt } from "@graphprotocol/graph-ts";
 
 export function handleNewAllocation(event: NewAllocationEvent): void {
+  _handleBackerStakingHistory(event);
+  _handleBacker(event);
+  _handleBackerToBuilder(event);
```

Comment:
Shouldn't we handle here `Builder.totalAllocation_`?

Reply:
I added but maybe we can have an additional discussion